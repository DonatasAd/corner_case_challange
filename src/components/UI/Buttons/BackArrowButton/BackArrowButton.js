import React from 'react';
import BackArrow from '../../../../assets/back-arrow.png';
import styles from './BackArrowButton.module.css';

export default function BackArrowButton(props) {
  return (
    <button {...props} className={styles.Button}>
      <img src={BackArrow} className={styles.Logo} alt='' />
    </button>
  );
}

BackArrowButton.defaultProps = {
  onClick: () => console.log('Back button'),
};
