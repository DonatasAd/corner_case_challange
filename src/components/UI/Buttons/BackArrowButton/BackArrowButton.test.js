import React from 'react';
import { shallow } from 'enzyme';
import BackArrowButton from './BackArrowButton';
describe('MyComponent', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<BackArrowButton debug />);
  
    expect(component).toMatchSnapshot();
  });
});