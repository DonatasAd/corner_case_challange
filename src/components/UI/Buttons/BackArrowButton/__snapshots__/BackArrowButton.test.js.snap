// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackArrowButton should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
